test/release/examples/primers/errorHandling.chpl

```diff
@@ -0,0 +1,143 @@
+// Error Handling
+//
+// This primer is a short tutorial on Chapel error handling. A detailed
+// outline can be found in the :ref:`readme-errorHandling` technote.
+//
+
+/*
+  .. _primers-errorHandling-handling:
+
+  Handling Errors
+  ---------------
+
+  Standard library functions that used to halt now throw errors for
+  developers to handle as they see fit. This is especially useful for
```

Comment:
We might not want to mention "used to halt" because that won't be relevant for long.
Maybe "Several important standard library functions throw errors so that developers
can handle the errors as they see fit"